test/elements.test.ts

```diff
@@ -1,22 +1,22 @@
 import { ElementDictionary } from "../src/data/elements";
 import { Phases, Phase } from "../src/interfaces/element-props";
 
-const elements: string[] = Object.keys(ElementDictionary);
-
 function checkUrl(url: string): boolean {
+  let isUrl: boolean = true;
```

Comment:
@qur786  if you are initializing the value no need to define the type.

Reply:
but isn't it robust and good practice to explicitly define the type?

Reply:
Yeah it is but when you initialize it it automatically takes the boolean type you can check it by doing `let isUrl = true;` and hovering over `isUrl` it will show boolean.

Reply:
So , change is needed?

Reply:
yeah